extension-readme.md

```diff
@@ -1,182 +1,242 @@
 # <a id="top"></a>AWS Toolkit for Visual Studio Code
 
-The AWS Toolkit for Visual Studio Code is an extension that enables you to interact with [Amazon Web Services (AWS)](https://aws.amazon.com/what-is-aws/) from within Visual Studio Code.
+The *AWS Toolkit for Visual Studio Code* is an extension that enables you to interact with [Amazon Web Services (AWS)](https://aws.amazon.com/what-is-aws/) from within the Visual Studio Code editor.
 
-*This is pre-release software and we recommend against using it in a production environment.*
+The following screenshots show important parts of the Toolkit.
+
+### <a id="ui-components"></a>Fundamental UI Components
 
 ![Overview](./resources/marketplace/overview.png)
 
-## Contents
+### <a id="open-command-palette"></a>AWS Commands in the Command Palette
+
+![Command Palette](./resources/marketplace/open-command-palette.gif)
```

Comment:
I am a little unsure if we need to show this...it's not explicitly AWS Toolkit instructions.

Reply:
It's not explicitly Toolkit, but it's integral to operating successfully, so I was thinking to address that need right away.
I think we should keep at least one of the call-outs (the static one in the previous picture or this one, if not both).

Reply:
My vote would be for this over the static one.

Reply:
Done.